src/LibraryManager/Providers/Unpkg/NpmPackageSearch.cs

```diff
@@ -18,16 +18,19 @@ internal class NpmPackageSearch
 
         public static async Task<IEnumerable<string>> GetPackageNamesAsync(string searchTerm, CancellationToken cancellationToken)
         {
+            if (searchTerm == null)
+            {
+                return new string[0];
+            }
+
             if (searchTerm.StartsWith("@"))
```

Comment:
nit: This could be an `else if`.

Reply:
Changed. Is the purpose of this change to make code looks cleaner?

---
In reply to: [195601754](https://github.com/aspnet/LibraryManager/pull/165#discussion_r195601754) [](ancestors = 195601754)